lstchain/high_level/tests/test_interpolate.py

```diff
@@ -0,0 +1,359 @@
+import astropy.units as u
+import numpy as np
+from astropy.io import fits
+from astropy.table import Table
+
+
+def test_compare_irfs(
+    simulated_irf_file, simulated_srcdep_irf_file, simulated_dl2_file,
+):
+    from lstchain.high_level.interpolate import compare_irfs
+    from lstchain.scripts.tests.test_lstchain_scripts import run_program
+
+    # Creating IRFs with different global cuts, en-dep cuts for point-like
+    # IRFs, to make the comparisons
+    irf_file_2 = simulated_dl2_file.parent / "diff_cut_irf.fits.gz"
+    run_program(
+        "lstchain_create_irf_files",
+        "--input-gamma-dl2",
+        simulated_dl2_file,
+        "--input-proton-dl2",
+        simulated_dl2_file,
+        "--input-electron-dl2",
+        simulated_dl2_file,
+        "--output-irf-file",
+        irf_file_2,
+        "--global-gh-cut=0.7",
+    )
+    irf_file_3 = simulated_dl2_file.parent / "en_dep_cut_irf_1.fits.gz"
+    run_program(
+        "lstchain_create_irf_files",
+        "--input-gamma-dl2",
+        simulated_dl2_file,
+        "--output-irf-file",
+        irf_file_3,
+        "--energy-dependent-gh",
+        "--energy-dependent-theta",
+        "--point-like",
+        "--gh-efficiency=0.7",
+        "--theta-containment=0.7",
+    )
+    irf_file_4 = simulated_dl2_file.parent / "en_dep_cut_irf_2.fits.gz"
+    run_program(
+        "lstchain_create_irf_files",
+        "--input-gamma-dl2",
+        simulated_dl2_file,
+        "--output-irf-file",
+        irf_file_4,
+        "--energy-dependent-gh",
+        "--energy-dependent-theta",
+        "--point-like",
+        "--gh-efficiency=0.8",
+        "--theta-containment=0.8",
+    )
+
+    irfs_1 = [simulated_irf_file, irf_file_2]
+    irfs_2 = [simulated_irf_file, simulated_irf_file]
+    irfs_3 = [irf_file_3, irf_file_4]
+    irfs_4 = [irf_file_3, irf_file_3]
+    irfs_5 = [simulated_srcdep_irf_file, simulated_srcdep_irf_file]
+
+    assert compare_irfs(irfs_1) == 0
+    assert compare_irfs(irfs_2)
+    assert compare_irfs(irfs_3) == 0
+    assert compare_irfs(irfs_4)
+    assert compare_irfs(irfs_5)
+
```

Comment:
I think you lack the comparison of two different irf files that should be equivalent. Here you only get true values for comparison of two of the same file.

Reply:
While this is not critical, your main use of the function is to check that two different original files are compatible (right?). So if you can implement an associated test with small efforts I would go for it.